Add subsection with frequent bloggers to NativeScript resources

RubSmeets opened the pull request:
Hi,

It could be useful to add some frequent NativeScript Bloggers to the list of resources. People who want to stay up-to-date on the latest developments can add the links to their RSS readers.

Kind Regards,
Ruben

joshwiens commented:
Something that was required to get [awesome-webpack](https://github.com/d3viant0ne/awesome-webpack) added to the master list was to add a little bio when highlighting people directly.

The path I took was to add a simple bio from information I found on the web and contact those listed after the fact and give them the opportunity to modify what is in their bio ( about half of them did ).

For reference - https://github.com/d3viant0ne/awesome-webpack#twitter

RubSmeets commented:
I did some more research on the NativeScript bloggers out there and came across the [nsbloggers](http://www.nsbloggers.com/) website. This site provides feeds of 15 active NativeScript bloggers which included some of my entries. For this reason I deleted two of my entries that where also listed on the [nsbloggers](http://www.nsbloggers.com/) website.

However, the feed of Peter Messenger's blog wasn't present on the website. I left the entry in the resources list and added a small Bio. I'm currently waiting on a reply from Peter on his Bio.

Kind regards
Ruben
